docs/api-reference/layer.md

```diff
@@ -476,6 +476,17 @@ The state object allows a layer to store persistent information cross rendering
 
 ### General Methods
 
+##### `clone`
+
+```js
+const updatedLayer = layer.clone(overrideProps);
+```
+
+Create a copy of this layer, optionally change some prop values.
```

Comment:
what is the cost of this call? Is it expensive?

Reply:
Should be similar to calling `new Layer({...})`